l10n_es_ticketbai_api/security/l10n_es_ticketbai_security.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2021 Binovo IT Human Project SL
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="l10n_es_tbai_invoice_rule" model="ir.rule">
+        <field name="name">Ticketbai Invoice Multi-Company</field>
+        <field name="model_id" ref="l10n_es_ticketbai_api.model_tbai_invoice" />
+        <field eval="True" name="global" />
+        <field
+            name="domain_force"
+        >['|', ('company_id', '=', False), ('company_id', 'child_of', [user.company_id.id])]</field>
```

Comment:
Esto no va así en v13. Tenéis que leeros la guía de migración:

https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0

Reply:
Entiendo que también habría que agregar el script de migración?

Reply:
Sí